### PR TITLE
feat-list

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (74)

### CODE REVIEW

File: comp-layouts/list.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <div class="list-container">
+       <div class="basic-list">
+     <ul>
+          <li>AVTAR</li>
+          <li>ALERT</li>
+          <li>IMAGE</li>
+          <li>FEAT</li>
+          <li>CODE</li>  
+</div>
+<div class="basic-list">
```
Comment (Sameony) [inline comp-layouts/list.html:23]: use prettier or beautify to realign the code

**Review by Sameony — APPROVED** [review]
clear code, but do pay mind to the proper alignment and refactoring of the code.

File: comp-layouts/list.html (hunk 1 above)
Comment (iamsaurav28) [reply comp-layouts/list.html:23]: ok i ll do it

**Review by therajatg — APPROVED** [review]
looks good to merge

**Pull request merged**